venues/MIDL.io/2019/Conference/webfield/programchairAbstractWebfield.js

```diff
@@ -0,0 +1,796 @@
+
+// Assumes the following pattern for meta reviews and official reviews:
+// CONFERENCE + '/-/Paper' + number + '/Meta_Review'
+// CONFERENCE + '/-/Paper' + number + '/Official_Review'
+
+// Constants
+var HEADER_TEXT = 'Program Chairs Console';
```

Comment:
Is there a reason you need to user another pc console than the one in the builder?

Reply:
I'll remove this file! Yay.